<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorWindow</class>
 <widget class="QMainWindow" name="DoctorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>575</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lekarz</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>20</y>
      <width>131</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dzisiejsi pacjenci</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="PatientsList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Wybrany pacjent:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="CurrentPatientName">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NextPatientButton">
       <property name="text">
        <string>NastÄ™pny pacjent</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>511</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Historia medyczna pacjenta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>Rok</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Rodzaj</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Nowe skierowanie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Dodaj wyniki</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Nowa recepta</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>511</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Dane pacjenta</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>27</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="rowWrapPolicy">
          <enum>QFormLayout::DontWrapRows</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Imie i nazwisko</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="PatientName">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Pesel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="PatientPesel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Adres</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="PatientAddress">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Przepisane leki</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="PatientMeds"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PatientMeds</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>DoctorWindow</receiver>
   <slot>show_prescribtion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NextPatientButton</sender>
   <signal>clicked()</signal>
   <receiver>DoctorWindow</receiver>
   <slot>get_next_patient()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DoctorWindow</receiver>
   <slot>new_referal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DoctorWindow</receiver>
   <slot>new_prescribtion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>DoctorWindow</receiver>
   <slot>show_history()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>DoctorWindow</receiver>
   <slot>add_results()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_prescribtion()</slot>
  <slot>choose_patient()</slot>
  <slot>get_next_patient()</slot>
  <slot>new_referal()</slot>
  <slot>new_prescribtion()</slot>
  <slot>show_history()</slot>
  <slot>add_results()</slot>
 </slots>
</ui>
